services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -150,9 +151,16 @@ testsPublic s =
             [ test s "happy flow" testInWhitelist,
               test s "handshake between LH device and user with old clients is blocked" testOldClientsBlockDeviceHandshake,
               testGroup "no-consent" $
-                flip fmap [(a, b, c, d) | a <- [minBound ..], b <- [minBound ..], c <- [minBound ..], d <- [minBound ..]] $
-                  \args@(a, b, c, d) ->
-                    test s (show args) $ testNoConsentBlockOne2OneConv a b c d,
+                ( ( (,,,)
```

Comment:
Nitpick: can we simplify this kind of thing to be easier to read / understand at a glance?

Reply:
Done. We used a list monad